### PR TITLE
Improve composer.json descriptions

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | 4.4 
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Refs #39768 
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: src/Symfony/Bridge/ProxyManager/composer.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "symfony/proxy-manager-bridge",
     "type": "symfony-bridge",
-    "description": "Symfony ProxyManager Bridge",
+    "description": "Provides integration for [ProxyManager][1] with various Symfony components",
```
Comment (xabbuh) [inline src/Symfony/Bridge/ProxyManager/composer.json]: Provides integration for ProxyManager with various Symfony components

File: src/Symfony/Bundle/DebugBundle/composer.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "symfony/debug-bundle",
     "type": "symfony-bundle",
-    "description": "Symfony DebugBundle",
+    "description": "Provides a tight integration between the Symfony Debug component and the Symfony full-stack framework",
```
Comment (xabbuh) [inline src/Symfony/Bundle/DebugBundle/composer.json]: "Provides a tight integration of the Symfony Debug component into the Symfony full-stack framework" (same wording as used for the SecurityBundle)

File: src/Symfony/Bundle/TwigBundle/README.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 TwigBundle
 ==========
 
+TwigBundle provides a tight integration between Twig and Symfony.
```
Comment (xabbuh) [inline src/Symfony/Bundle/TwigBundle/README.md]: "TwigBundle provides a tight integration of Twig into the Symfony full-stack framework" (same wording as used for the SecurityBundle)

File: src/Symfony/Bundle/TwigBundle/composer.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "symfony/twig-bundle",
     "type": "symfony-bundle",
-    "description": "Symfony TwigBundle",
+    "description": "Provides a tight integration between Twig and Symfony",
```
Comment (xabbuh) [inline src/Symfony/Bundle/TwigBundle/composer.json]: "Provides a tight integration of Twig into the Symfony full-stack framework" (same wording as used for the SecurityBundle)

File: src/Symfony/Bundle/WebProfilerBundle/composer.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "symfony/web-profiler-bundle",
     "type": "symfony-bundle",
-    "description": "Symfony WebProfilerBundle",
+    "description": "Provides a **development tool** that gives detailed information about the execution of any request",
```
Comment (xabbuh) [inline src/Symfony/Bundle/WebProfilerBundle/composer.json]: s/**development tool**/development tool/

File: src/Symfony/Component/Messenger/README.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
 Messenger Component
 ===================
 
-The Messenger component helps application send and receive messages to/from other applications or via
-message queues.
+The Messenger component helps application send and receive messages to/from
```
Comment (xabbuh) [inline src/Symfony/Component/Messenger/README.md]: [...] helps applications send [...]

File: src/Symfony/Component/Messenger/composer.json [hunk 7]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "symfony/messenger",
     "type": "library",
-    "description": "Symfony Messenger Component",
+    "description": "Helps application send and receive messages to/from other applications or via message queues",
```
Comment (xabbuh) [inline src/Symfony/Component/Messenger/composer.json]: Helps applications send [...]

File: src/Symfony/Component/PropertyAccess/composer.json [hunk 8]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "symfony/property-access",
     "type": "library",
-    "description": "Symfony PropertyAccess Component",
+    "description": "Provides function to read and write from/to an object or array using a simple string notation",
```
Comment (xabbuh) [inline src/Symfony/Component/PropertyAccess/composer.json]: Provides functions to [...]

**Comment by fabpot** [comment]
@xabbuh All fixed, thanks for the review.

File: src/Symfony/Bridge/Monolog/README.md [hunk 9]
```diff
@@ -1,7 +1,8 @@
 Monolog Bridge
 ==============
 
-Provides integration for Monolog with various Symfony components.
+The Monolog bridge provides integration for Monolog with various Symfony
```
Comment (wouterj) [inline src/Symfony/Bridge/Monolog/README.md]:
```suggestion
The Monolog bridge provides integration for
[Monolog](https://seldaek.github.io/monolog/) with various Symfony
```

**Review by xabbuh — APPROVED** [review]

File: src/Symfony/Bridge/PhpUnit/README.md [hunk 10]
```diff
@@ -1,7 +1,8 @@
 PHPUnit Bridge
 ==============
 
-Provides utilities for PHPUnit, especially user deprecation notices management.
+The PHPUnit bridge provides utilities for PHPUnit, especially user deprecation
+notices management.
```
Comment (wouterj) [inline src/Symfony/Bridge/PhpUnit/README.md]:
```suggestion
The PHPUnit bridge provides utilities for [PHPUnit](https://phpunit.de/),
especially user deprecation notices management.
```

File: src/Symfony/Component/Config/README.md [hunk 11]
```diff
@@ -1,9 +1,9 @@
 Config Component
 ================
 
-The Config component provides several classes to help you find, load, combine,
-autofill and validate configuration values of any kind, whatever their source
-may be (YAML, XML, INI files, or for instance a database).
+The Config component helps find, load, combine, autofill and validate
```
Comment (chalasr) [inline src/Symfony/Component/Config/README.md:4]: `helps find` vs `helps you find` in composer description. To be synced?

**Review by chalasr — APPROVED** [review]

**Review by wouterj — APPROVED** [review]

**Review by derrabus — APPROVED** [review]

**Pull request merged**